### PR TITLE
Updating code to use auto-configuration

### LINKED ISSUE #46: Update javadsl sample
Related to https://github.com/spring-cloud/spring-cloud-dataflow/issues/1965

### PR DESCRIPTION
**Pull request by viniciusccarvalho**
Fixes #46

### CODE REVIEW

**Comment by viniciusccarvalho** [comment]
@markpollack I commented out the app import since I could not get it to work here, if you don't mind testing it on your end, and if it works there remove the comment (if no other reviews) during merge please

**Comment by markpollack** [comment]
Should we be removing the `.idea` director?

File: javadsl/src/main/java/com/example/scdfdsl/ScdfdslApplication.java [hunk 1]
```diff
@@ -52,8 +64,8 @@ public StreamApplication sink() {
 
 	@Override
 	public void run(ApplicationArguments applicationArguments) throws Exception {
-		DataFlowOperations dataFlowOperations = createDataFlowOperations(applicationArguments);
-
+//		dataFlowOperations.appRegistryOperations().importFromResource(
```
Comment (markpollack) [inline javadsl/src/main/java/com/example/scdfdsl/ScdfdslApplication.java]: Should decide if this line is in our out.

Comment (viniciusccarvalho) [reply javadsl/src/main/java/com/example/scdfdsl/ScdfdslApplication.java]: That was my original question in the comments

**Comment by sabbyanandan** [comment]
This is already merged.

**Pull request closed without merging**